spec/authorization/index.md

```diff
@@ -373,14 +373,14 @@ the FHIR authorization server, establishing the following
 
  * JWT "issuer" URL: `bili_monitor`
  * OAuth2 `client_id`: `bili_monitor`
- * JWK identfier: `kid` value (see [example JWK](https://github.com/smart-on-fhir/fhir-bulk-data-docs/blob/master/sample-jwks/RS384.public.json))
+ * JWK identfier: `kid` value (see [example JWK](https://raw.githubusercontent.com/HL7/bulk-data/master/resources/sample-jwks/RS384.public.json))
```

Comment:
This should link to content available directly in the official repo, rather than to the SMART repo which may disappear

Reply:
@ricky1uphealth use relative urls

Reply:
fixed here https://github.com/HL7/bulk-data/pull/10/commits/ad5cba4c18ff42427e5d3116f221deb7ef2dc044

Reply:
fixed here https://github.com/HL7/bulk-data/pull/10/commits/d0c3c2555cc1dc46942b000a702ba37d93155211